src/components/network-area-diagram-viewer/network-area-diagram-viewer.ts

```diff
@@ -1114,4 +1201,94 @@ export class NetworkAreaDiagramViewer {
             this.redrawVoltageLevelNode(otherNode, busNodeEdges, movedEdges);
         }
     }
+
+    private moveThreeWtEdge(
+        edge: SVGGraphicsElement,
+        edgeNode: SVGGraphicsElement,
+        offset: Point
+    ) {
+        const twtEdge: HTMLElement = <HTMLElement>edgeNode.firstElementChild;
+        if (twtEdge != null && twtEdge.tagName == 'polyline') {
+            const pointsAttributes = twtEdge.getAttribute('points');
+            if (pointsAttributes != null) {
+                const points = DiagramUtils.getPolylinePoints(pointsAttributes);
+                if (points != null) {
+                    // compute polyline points
+                    const edgeNodes = this.getEdgeNodes(edge);
+                    const threeWtMoved =
+                        edgeNodes[1]?.id == this.selectedElement?.id;
+                    const busNodeId1 = edge.getAttribute('busnode1');
+                    const unknownBusNode1 =
+                        busNodeId1 != null && busNodeId1.length == 0;
+                    const nodeRadius1 = this.getNodeRadius(
+                        busNodeId1 != null ? +busNodeId1 : -1
+                    );
+                    const edgeStart = DiagramUtils.getDistance(
+                        DiagramUtils.getPosition(edgeNodes[0]),
+                        DiagramUtils.getPosition(edgeNodes[1]),
+                        unknownBusNode1
+                            ? nodeRadius1[1] +
+                                  this.svgParameters.getUnknownBusNodeExtraRadius()
+                            : nodeRadius1[1]
+                    );
```

Comment:
duplicate code with moveStraightEdge content. We could create a function to factorize this, and use it also for both sides of moveStraightEdge?

Reply:
Added method getEdgeStart